packages/data-point/lib/entity-types/entity-request/__snapshots__/resolve.test.js.snap

```diff
@@ -0,0 +1,23 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`resolveRequest log errors when request fails 1`] = `
+"404 - \\"not found\\"
+
+Entity info:
+  Id: test:test
+  - options: { json: true, url: 'http://remote.test/source1' }
+  - value: 'foo'
```

Comment:
It'd be easier to parse this if the format were consistent for each "bullet":

```javascript
Id: test:test
Options: { json: true, url: 'http://remote.test/source1' }
Value: 'foo'
```

Also, if the options get really large having a multiline object might be easier to read:

```javascript
Id: test:test
Options: {
  json: true,
  url: 'http://remote.test/source1'
}
Value: 'foo'
```

Reply:
agree with adding the bullet, which btw I plan to change on a second iteration once we add **chalk** and colorize all of these strings